### PR TITLE
Made stats controller

### PR DESCRIPTION
**Pull request by adamsr123**

### CODE REVIEW

File: controllers/stats/deskStats.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const _ = require('lodash');
```
Comment (nachtiomer) [inline controllers/stats/deskStats.js:1]: this file is not a contoller!
controller should take care of api requests
move it to utils/helpers folders

File: controllers/stats/missionStats.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const _ = require('lodash');
```
Comment (nachtiomer) [inline controllers/stats/missionStats.js:1]: again, not a controller. move it

File: controllers/stats/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const { Router } = require('express');
```
Comment (nachtiomer) [inline controllers/stats/index.js:1]: change it to stats.js directly under controllers

File: controllers/stats/statuses.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
+const USER_TYPES = require('./userTypes');
```
Comment (nachtiomer) [inline controllers/stats/statuses.js:1]: not a controller

File: index.js [hunk 5]
```diff
@@ -11,6 +11,8 @@ const PORT = 8080;
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
+app.use(require("morgan")("tiny"))
```
Comment (nachtiomer) [inline index.js:14]: what is that?

Comment (adamsr123) [reply index.js:14]: this is a very useful and great package for logging all requests! helps a lot to look at the console while working

File: config.js [hunk 6]
```diff
@@ -13,4 +13,4 @@ const Config = {
     }
 }
 
-module.exports = Config[process.env.NODE_ENV] || Config.development;
\ No newline at end of file
+module.exports = Config.redesign || Config.development;
```
Comment (nachtiomer) [inline config.js:16]: what

Comment (adamsr123) [reply config.js:16]: lol by mistake

**Pull request merged**